qemu/tests/cfg/qemu_img.cfg

```diff
@@ -64,6 +64,17 @@
                             #cluster_size set cluster size used in qemu-img -o cluster_size=, default is 64k.
                             cluster_size = 2048
                             no Host_RHEL.m5
+                        - show_progress:
+                            command_result_pattern = "\(\d+\.\d+/100%\)"
+                            variants:
+                                - on:
```

Comment:
on/off is a general name, it's maybe duplicated in other test, it's better to set a special name.